styles.css

```diff
@@ -1,3 +1,4 @@
+
 .cards-preview .img-container{
```

Comment:
suggestion: add a space before { its a good practice and use beautify extension in vs code it makes all the spacing correct.

Reply:
Understood, thanks